ecosystem/indexer-grpc/indexer-transaction-generator/README.md

```diff
@@ -1,3 +1,21 @@
 # Indexer Transaction Generator
 
-This tool is to generate transactions for testing purpose.
\ No newline at end of file
+This tool is to generate transactions for testing purpose.
+
+## Usage
+
+`cargo run -- --config example.yaml --output-folder /your_path_to_store_transactions/`
+
+### Config
+
+```YAML
+import_config:
+  testnet_config:
```

Comment:
Should we only support this for testnet and mainnet networks? 

Importing devnet txn's is weird because the data gets wiped

Reply:
Yea current impl allows the flexibility without using devnet or custom network. 

The ability to import transaction from devnet allows users to verify their change quickly.